apache-solr/dashboards/apache-solr-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1680 @@
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'apache-solr-cluster-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local liveNodesPanel = {
+  datasource: {
+    type: 'prometheus',
+    uid: '${prometheus_datasource}',
+  },
+  description: 'Number of live nodes in the Solr cluster.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'red',
+            value: null,
+          },
+          {
+            color: 'green',
+            value: 1,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'value',
+    wideLayout: true,
+  },
+  pluginVersion: '9.4.3',
+  targets: [
+    {
+      disableTextWrap: false,
+      expr: 'min by (job, solr_cluster) (solr_collections_live_nodes{job=~"$job", solr_cluster=~"$solr_cluster"})',
+      fullMetaSearch: false,
+      includeNullMetadata: true,
+      instant: false,
+      legendFormat: '__auto',
+      range: true,
+      useBackend: false,
+    },
+  ],
+  title: 'Live nodes',
+  type: 'stat',
+};
+
+local zookeeperStatusPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'solr_zookeeper_status{job=~"$job", solr_cluster=~"$solr_cluster"}',
+      datasource=promDatasource,
+      intervalFactor=2,
+      instant=true,
+      legendFormat='{{zk_host}}',
+      format='table',
+    ),
+  ],
+  type: 'table',
+  title: 'Zookeeper status',
```

Comment:
Would it make sense to make these drill down? Im not quite sure how the architecture behind solr works so I dont know whether drilling down into these instances would take you into one of the other dashboards, or whether zookeeper is completely tangent to solr and this is better as left as is.

Reply:
The other dashboards have no related metrics to Zookeeper. The Zookeeper metrics help different solr instances communicate with each other which is why it is on the cluster overview dashboard.